README.md

```diff
@@ -13,6 +13,7 @@ A set of examples to help demontrate common configurations using Mocha. They sho
 * [Third party reporter](third-party-reporter/)
 * [TypeScript application](typescript/)
 * [Vue and Puppeteer application](vue-puppeteer/)
+* [TypeScript + Babel combination](typescript-babel-combination/)
```

Comment:
Could you rename folder `typescript-babel` pls? Be good to get a consistency going with them. Same for here i.e. `TypeScript, Babel`.